projects/novo-elements/src/elements/progress/ProgressBar.ts

```diff
@@ -97,7 +97,12 @@ export class NovoProgressBarElement implements ControlValueAccessor, OnInit {
     if (this.value !== value) {
       this._value = value;
       if (this.progress) {
-        this._percent = this.value / this.progress.total;
+        if (this.progress.total > 0) {
+          this._percent = this.value / this.progress.total;
+        } else {
+          this._percent = 0;
```

Comment:
If these changes are the exact same change in two places, we could either take the time to make this a single method, so that there's no duplication (Don't repeat yourself principle) or you could keep these changes to a single line, in which you don't need a method. 

```
this._percent = this.progress.total > 0 ? this._value / this.progress.total : 0;
```

Reply:
Very true, fixed this